packages/next/client/image.tsx

```diff
@@ -209,6 +219,24 @@ function defaultImageLoader(loaderProps: ImageLoaderProps) {
   )
 }
 
+function removePlaceholder(
+  element: HTMLImageElement | null,
+  placeholder: PlaceholderValue
+) {
+  const hasBlurryPlaceholder =
+    configEnableBlurryPlaceholder && placeholder === 'blur'
+  if (hasBlurryPlaceholder && element) {
```

Comment:
```suggestion
  if (placeholder === 'blur' && element) {
```

No longer need to check config

Reply:
Thanks for the catch, done.